extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -141,6 +143,7 @@ For example
 * If `offset` is 0 and `repeat` is 2, the feature IDs are `[0, 0, 1, 1, 2, 2, ...]`
 * If `offset` is 2 and `repeat` is 3, the feature IDs are `[2, 2, 2, 3, 3, 3, 4, 4, 4, ...]`
 * If `offset` is 2 and `repeat` is undefined, the feature IDs are `[2, 2, 2, ...]`
+> **Implementation note:** When rasterizing a primitive with feature ID attributes, points in the interior of a triangle should be considered to belong to the feature associated with the nearest vertex.
```

Comment:
I'm not sure if we should duplicate the implementation note. I'm fine with it in either spot.

Reply:
put it in the first spot